packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react'
+import {
+  ComponentSlotStylesInput,
+  ICSSInJSStyle,
+  SiteVariablesPrepared,
+  NaturalColors,
+} from '../../../types'
+import { AlertProps } from '../../../../components/Alert/Alert'
+import { AlertVariables } from './alertVariables'
+
+const getIntentColorsFromProps = (
+  p: AlertProps,
+  v: AlertVariables,
+  siteVars: SiteVariablesPrepared,
+): React.CSSProperties => {
+  const { colors } = siteVars
+  const naturalColors: NaturalColors = siteVars.naturalColors
+
+  if (p.danger) {
+    return {
+      color: siteVars.red,
+      backgroundColor: siteVars.red10,
+      borderColor: siteVars.red08,
```

Comment:
Maybe we need to update the values here, as the `red08` was changed and it shouldn't be.

Reply:
here it's correct, the problem is in `siteVariables` only, thanks again for catching it.